Resources/Prototypes/Entities/Objects/Consumable/drinks.yml

```diff
@@ -57,6 +57,18 @@
     damage:
       types:
         Blunt: 5
+  - type: FitsInDispenser
```

Comment:
The main idea behind FitsInDispenser is that you can't fit a jug in Dispenser. 

Since WaterJug inherits a `DrinkGlassBase` it's going to fit in Dispenser. Also, you can put various non-beaker stuff and fill them with reagents. Like Water cups and beer bottles. 

And the moment you allow beer bottles, people will complain that the dispenser doesn't reflect the color of the beer bottle glass.

Reply:
Keep in mind the soda dispenser (a reagent dispenser) being effectively useless (unable to accept an empty drinking glass) is what caused this part of the PR.

Reply:
I guess compromise solution would be: make jug and other large holders not fit in dispenser.

Reply:
So as it turns out, TransformableContainers are *supposed to* always fit in dispensers: https://github.com/space-wizards/space-station-14/blob/master/Content.Server/Chemistry/Components/TransformableContainerComponent.cs
So either jugs/etc. can't be transformable container targets, or everything has to fit in a dispenser.